kobo/client/__init__.py

```diff
@@ -129,7 +129,8 @@ class ClientCommand(kobo.cli.Command):
 class HubProxy(object):
     """A Hub client (thin ServerProxy wrapper)."""
 
-    def __init__(self, conf, client_type=None, logger=None, transport=None, auto_logout=None, **kwargs):
+    def __init__(self, conf, client_type=None, logger=None, auto_logout=None,
```

Comment:
This change reorders some positional arguments.  That's backwards-incompatible, can you please avoid that?

For example, if someone has written:

```
proxy = HubProxy(conf, "worker", logger, None, True)
```

Prior to this change, that `True` meant `auto_logout=True`.
When they upgrade to the kobo version with this change, it will instead mean `transport=True` and their code will crash.

We might hope that everyone has only used named arguments here, but it's not something we can assume.